internal/dag/gatewayapi_processor.go

```diff
@@ -113,11 +125,81 @@ func (p *GatewayAPIProcessor) Run(dag *DAG, source *KubernetesCache) {
 
 		// Process all the routes that match this Gateway.
 		for _, matchingRoute := range matchingRoutes {
-			p.computeHTTPRoute(matchingRoute)
+			p.computeHTTPRoute(matchingRoute, listenerSecret)
 		}
 	}
 }
 
+func (p *GatewayAPIProcessor) validGatewayTLS(listener gatewayapi_v1alpha1.Listener) *Secret {
+
+	// Validate the CertificateRef is configured.
+	if listener.TLS.CertificateRef == nil {
+		p.Error(contour_api_v1.ConditionTypeTLSError, "SecretNotConfigured",
```

Comment:
This and the other log lines in this function confused me for a sec because they _look_ like `Conditions`, but they're actually logging -- is this format for the logging desired, or would it make sense to switch to more standard `.Errorf(...)` formatting?

Reply:
I cleaned up, I think what happened was I was writing status to the Gateway, then turned out that Contour wasn't going to do that, so it got left like this.